package.json

```
@@ -2,9 +2,11 @@
   "name": "nats.ws",
   "version": "1.0.1",
   "description": "WebSocket NATS client",
-  "main": "nats.mjs",
+  "main": "nats.js",
+  "module": "nats.mjs",
```

I think you should also add the `exports` property which is the latest standard for defining CJS and ESM flavors, see https://docs.skypack.dev/package-authors/package-checks#export-map


```suggestion
  "module": "nats.mjs",
  "exports": {
    ".": {
      "require":  "./nats.js",
      "default": "./nats.mjs"
    }
  },
```